extensions/default/src/MergeDataSource/index.js

```diff
@@ -0,0 +1,195 @@
+import { DicomMetadataStore, IWebApiDataSource } from '@ohif/core';
```

Comment:
Could you move this to typescript - it would make figuring out a few of the calls a bit easier, as I wasn't always sure of the types.

Reply:
Updated, thanks.